scripts/yggdrasil-iptunnel/yggdrasil-setup

```diff
@@ -4,36 +4,137 @@
 echo 1 > /proc/sys/net/ipv4/ip_forward
 echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
 
-SUBNET4="10.10.0."
+function confset {
```

Comment:
This should probably be in `functions.sh`.

Reply:
Yes, for testing only but will make its way out of into a shared location after

Reply:
Why not now? It's fine with me here if it moves eventually (before master?), but I would do that personally. If it needs testing that should be done before it gets merged.

Reply:
it will be one of the first things i do in after this merge because it is needed for other issues to be completed.

But as of now it works and i dont want to "break" it by just moving it. Id rather do it when im testing stuff.